src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -298,8 +302,9 @@ private ParseUserDefinitionResult ParseUDFsAndNamedFormulas(string script, Parse
 
                 if (_curs.TidCur == TokKind.Equ)
                 {
+                    var declaration = script.Substring(declarationStart, _curs.TokCur.Span.Min - declarationStart);

```

Comment:
why substring rather than using the identifier token?

Reply:
This is storing everything up to the `=` (or `{`) to easily preserve any comments.